docs/usage/getting-started/private-packages.md

```diff
@@ -121,9 +121,11 @@ The following details the most common/popular manager artifacts updating and how
 
 ### composer
 
-Any `hostRules` token for `github.com` or `gitlab.com` are found and written out to `COMPOSER_AUTH` in env for Composer to parse.
+Any `hostRules` token for `github.com` or `gitlab.com` are found and written out to `COMPOSER_AUTH` in env for Composer to parse. For `github.com`, `hostType=git-tags` can be used to narrow down the use of the PAT for this use case only.
```

Comment:
```suggestion
Any `hostRules` token for `github.com` or `gitlab.com` are found and written out to `COMPOSER_AUTH` in env for Composer to parse.
For `github.com`, `hostType=git-tags` can be used to narrow down the use of the PAT for this use case only.
```

I think we need to find a better wording than "can be used". Does that mean the user is meant to do something, or just to explain what we have done?

Reply:
For github,  most integrations are better with the default renovate application token, that's why we cannot juste add a simple hostRule for github.com and a personel access token.

The only problem is composer that require a personnal access token.  I used the `git-tags` hostRule because it's the same rule that is used for packagist dependency lookup.

Maybe we should be more clear about this in the documentation. Do you have a suggestion ?